Makefile

```diff
@@ -123,6 +131,17 @@ $(OC) oc: $(TOOLS_DIR)
 		$(OC) version | grep -q $${version##v} ;\
 	}
 
+.PHONY: jsonnet-tools
+$(JSONNET) jsonnet-tools: $(TOOLS_DIR)
```

Comment:
This is bit different to the installation of other tools and I think either
* we remove $(JSONNET) and any target that requries $(JSONNET*) should depend on `jsonnet-tools`
* or list all the tools that are getting installed as targets

Reply:
I will remove `$(JSONNET)` and depend on `jsonnet-tools` in targets requiring this